canonical-web-and-design/ubuntu.com
# Update reduced nav and secondary nav

## Done

- Removed global nav 
- Updated the current nav to be reduced 
- Updated secondary nav

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Demo at https://ubuntu-com-11497.demos.haus/

## Issue / Card

Fixes [#5107](https://github.com/canonical-web-and-design/web-squad/issues/5107), [#5072](https://github.com/canonical-web-and-design/web-squad/issues/5072)

## Screenshots
<img width="1427" alt="image" src="https://user-images.githubusercontent.com/57550290/164182694-06efe7d1-ee0e-4828-bb50-d69e8f734ca0.png">


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
